packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -8,7 +8,13 @@ import { get, escapeRegExp } from 'lodash';
 let language = 'en-US';
 let window = null;
 let stub = null;
-const randomFileName = `fileName${Math.random() * 10000}`;
+export const randomFileName = `fileName${Math.random() * 10000}`;
+
+const doesDocumentContainText = (text: string) => {
```

Comment:
nit: best practice for cypress would be create a cy command if it is used across

Reply:
updating in a follow-up PR